linkerd/admin/src/main/resources/io/buoyant/linkerd/admin/template/router_client.template

```diff
@@ -2,27 +2,58 @@
   {{client}}
 </div>
 
-<div class="client-metrics">
-  <dl>
-  {{#each metrics}}
+<div class="client-metrics col-md-4">
+  <div class="row">
     <div class="col-md-2">
-      <dt>
-        {{description}}
-      </dt>
+      <dt>Requests</dt>
+      <dd>{{requests}}</dd>
+    </div>
+    <div class="col-md-2">
+      <dt>Connections</dt>
+      <dd>{{connections}}</dd>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-md-2">
+      <dt>Successes</dt>
+      <dd>
+        {{#if successRate}}
+          {{successRate}}
+        {{else}}
+          {{success}}
+        {{/if}}
+      </dd>
+    </div>
+    <div class="col-md-2">
+      <dt>Failures</dt>
       <dd>
-        <span class="metric-medium">
-          {{#if rate}}
-            {{rate}}
-          {{else}}
-            {{#if value}}
-              {{value}}
-            {{else}}
-              0
-            {{/if}}
-          {{/if}}
-        </span>
+        {{#if failures}}
+          {{failures}}
+        {{else}}
+          0
+        {{/if}}
       </dd>
     </div>
-  {{/each}}
-  </dl>
+  </div>
+</div>
+
+<div class="col-md-2">
+  <div>Latencies</div>
+  <table class="router-latency-table">
```

Comment:
These stats _barely_ fit the definition of what I might consider tabular data, so I'll let it slide. Know that generally going forward I'd like to avoid using tables (I honestly don't care for data lists much either).  It makes separation of content & design much easier if they're not used